### PR TITLE
Create .gitattributes and manifest

### PR DESCRIPTION
**Pull request by mavaylon1**

### CODE REVIEW

File: MANIFEST.in [hunk 1]
```diff
@@ -0,0 +1,4 @@
+include license.txt Legal.txt versioneer.py src/hdmf_zarr/_version.py src/hdmf_zarr/_due.py
```
Comment (rly) [inline MANIFEST.in]: Is this case-sensitive? The repo contains `License.txt`. It also does not contain `Legal.txt`. I am not sure if that matters. Can you try creating a wheel and distribution to see if these files get packaged correctly?

Comment (mavaylon1) [reply MANIFEST.in]: I believe it's case insensitive. I updated it either way.

Comment (mavaylon1) [reply MANIFEST.in]: I did a test with creating a wheel and it is case insensitive.

**Comment by rly** [comment]
Note that in your PR text, you can say "Fix #23", and the issue will be linked to from here.

**Comment by rly** [comment]
And the issue will be automatically closed when this PR is merged

**Review by rly — APPROVED** [review]

**Pull request merged**